frontend/express/public/stylesheets/styles/manifest.scss

```diff
@@ -37,4 +37,4 @@
 @use "../../core/session-durations/stylesheets/main" as session-durations-main-style;
 @use "../../core/session-frequency/stylesheets/main" as session-frequency-main-style;
 @use "../../core/events/stylesheets/main" as events-overview-main-style;
-
+@use "../../core/user-management/stylesheets/main" as user-management-main-style;
```

Comment:
Do we really need this in manifest? @itsiprikshit

Reply:
yes we do.
bcz the css for core features was initially in the main.css
but with sass we create partials _main.scss for each core feature.
and if they are not imported in the manifest, the css wont be compiled.

Reply:
we can come up with a different method, but this was the simplest for now.